packages/callout/src/Callout.story.tsx

```diff
@@ -64,3 +67,16 @@ WithRichContent.args = {
     </>
   ),
 };
+
+export const WithLinks = Template.bind({});
+WithLinks.args = {
+  title: 'Title',
+  children: (
+    <>
+      Lorem Ipsum is simply dummy text &nbsp;
+      <Link href="http://localhost:9001">Link component</Link>
+      &nbsp;
+      <a href="http://localhost:9001">Regular link</a>
+    </>
+  ),
```

Comment:
Just a mental note... I'm starting to notice this could become a reusable block as well maybe?? Or maybe that's going overboard. Just a thought.

Reply:
I don't really see the need to make them reusable for now since they are simple links but its good to keep in mind.